src/core/a-node.js

```diff
@@ -128,6 +129,13 @@ module.exports = registerElement('a-node', {
           self.emit('loaded', undefined, false);
         }).catch(function (err) {
           error('Failure loading node: ', err);
+
+          // Failed to load a node, but things won't get any better by waiting around.
+          // So get on with rendering the scene by signaling that this node has loaded.
+          // An "error" event has already been fired, so we don't need to fire another one.
+          warn('Rendering scene with errors on node: ', self);
```

Comment:
Don't you think this can be misleading? Worried that we're masking errors that the dev might want to stop and fix.

Reply:
There's already a console error about the failure to load a file.
So it didn't seem to me like another error was helpful, and a warning was the appropriate error level here.

Is there anything we can do apart from console warnings & errors to draw a Dev's attention to the fact that something is not right?

(apart from failing to render the scene at all, which as per #5032 I don't think that's a good solution).